### ISSUE TITLE
NoMethodError: undefined method `where'

### ISSUE THREAD

**Issue by pusewicz** [biola/punching_bag#27]
Hey @adamcrown!

Both calls to `uniq` should be replaced with `distinct` in new versions of Rails.

https://github.com/biola/punching_bag/blob/d7f7a2905feedbf0e37140d283a9530f288d63ee/lib/tasks/punching_bag.rake#L6-L10

Otherwise:

```
rake aborted!
NoMethodError: undefined method `where' for #<Array:0x00007fe2cd8d1d70>
/Users/piotr/.rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/punching_bag-0.6.0/lib/tasks/punching_bag.rake:10:in `block (3 levels) in <main>'
```

**Comment by Dvelezs94** [comment]
hey @pusewicz do you have a branch or something where  we can download the fix? Thanks in advance

**Comment by Dvelezs94** [comment]
@pusewicz do you recommend any other alternative? I was looking at [impressionist](https://github.com/charlotte-ruby/impressionist) and it looks very similar, but I dont see any job that joins the rows by dates like punching bag

**Comment by Dvelezs94** [comment]
well if anybody else is encountering this issue, i created a fork here https://github.com/Dvelezs94/punching_bag, you can just add it to your Gemfile like this `gem 'punching_bag', git: 'https://github.com/Dvelezs94/punching_bag'` I will also be accepting PRs actively if you encounter any issue

**Comment by adamcrown** [comment]
Hi @Dvelezs94. I apologize for how very long it's taken me to get to this issue. I recently switched from a very demanding job to a job with a much more normal work load, and I'm making my way back through issues and pull requests slowly.

If you would like to open a pull request from your fork, I'd be happy to look at merging it. In the meantime, I'll see if I can get the issue mentioned above fixed. Thanks.

**Comment by adamcrown** [comment]
I just released version `0.6.1` that should fix this issue. Thanks for reporting it.
